packages/cli/src/handlers/compile.ts

```diff
@@ -37,6 +39,23 @@ export const compile = async (options: GenerateHandlerOptions) => {
         properties: {},
     });
 
+    const dbtVersion = await getDbtVersion();
+    GlobalState.debug(`> DBT version ${dbtVersion}`);
+
+    if (!dbtVersion.includes('1.3.') && process.env.CI !== 'true') {
+        const answers = await inquirer.prompt([
+            {
+                type: 'confirm',
+                name: 'isConfirm',
+                message: `${styles.warning(
+                    `Your DBT version ${dbtVersion} does not match our supported version (1.3.0), this could cause problems on compile or validation.`,
```

Comment:
I think we should show this message ( not the question ) in the CI as well. For debugging purposes.